lms/djangoapps/grades/tasks.py

```diff
@@ -16,7 +16,6 @@
 from opaque_keys.edx.keys import UsageKey
 from opaque_keys.edx.locator import CourseLocator
 from submissions import api as sub_api
-from student.models import anonymous_id_for_user
```

Comment:
unused import - yes - got squeezed in - didn't seem worth it to have a completely separate PR for it.